### PR TITLE
Add support to show upload progress during release in CLI

### PR DESCRIPTION
**Pull request by shishirx34**
Also added a gulp task to clean CLI and SDK.
Superagent recently added support to listen to the on 'progress' events. Unfortunately it hasn't yet been published to NPM, therefore pointing the superagent node module to a commit that has our required changes. We can probably move it off to npm once we get a superagent@1.5.0 update.

### CODE REVIEW

**Comment by lostintangent** [comment]
LGTM!

File: sdk/script/management/account-manager.ts [hunk 1]
```diff
@@ -816,8 +816,14 @@ export class AccountManager {
                 file = fileOrPath;
             }
 
-            req.field("package", file)
+            req.attach("package", file)
                 .field("packageInfo", JSON.stringify(packageInfo))
+                .on("progress", (event: any) => {
+                    if (uploadProgressCallback) {
+                        var currentProgress: number = (event.loaded/event.total) * 100;
```
Comment (dtivel) [inline sdk/script/management/account-manager.ts]: Remove superfluous parentheses.
Fix whitespace around '/'.
Can "event.total" ever be 0?  Perhaps when the package is an empty file.  If yes, the result would be Infinity.

**Comment by dtivel** [comment]
Looks good.

**Comment by shishirx34** [comment]
This is how it looks.

image

**Pull request merged**